react-backend/shared/controllers/feedback.js

```diff
@@ -0,0 +1,24 @@
+const jwt = require('jsonwebtoken');
+const cbAdmin = require('../models/cb-admin');
+
+module.exports = (req, res, next) => {
+  const pgPool = req.app.get('client:psql');
+  const standardJwtSecret = req.app.get('cfg').session.standard_jwt_secret;
+  const { feedbackScore } = req.body;
+  const { authorization: token } = req.headers;
+
+  jwt.verify(token, standardJwtSecret, (err, decoded) => {
+    if (err) return next;
+
+    const { email: cbEmail } = decoded;
+
+    cbAdmin
+      .insertFeedback(pgPool, { cbEmail, feedbackScore })
+      .then(data => res.send({ result: data }))
+      .catch(() =>
+        res
```

Comment:
This `catch` will catch _any_ error occurring during the promise chain. It should be used for errors that we cannot anticipate and/or mitigate in advance (like programmer errors).

We _can_ anticipate payload validation errors. You're not doing any validation in this controller. Then you can correctly respond with a `400` for payload errors, and leave this as a `500`.